form-val.js

```diff
@@ -10,3 +10,23 @@ function validate(e) {
     err.innerHTML = 'Email cannot be uppercase';
   }
 }
+
+const name = document.getElementById('#name');
+const email = document.getElementById('#email');
+const text = document.getElementById('#message');
+
+let contactObj = {
+  name: 'mike',
+  email: 'madetunji1376@gmail.com',
+  text: 'i love football',
+};
+
+localStorage.setItem('info', JSON.stringify(contactObj));
+
+let infoString = localStorage.getItem('info');
+console.log('string saved in localStorage', infoString);
+
+let savedContact = JSON.parse(infoString);
+console.log('person name:', savedContact.name);
+console.log('person email:', savedContact.email);
+console.log('person text:', savedContact.text);
```

Comment:
if i dont console...how do i make them populate?
by just removing them without console log?
@cwaku

Reply:
@Therealadetunji I am just seeing this, thanks for your patience. After you parse the data from the `localStorage`, instead of having to `console.log()` them, you could get the input fields from the `DOM` and then populate their values.
A simple example would be:
```
let name = document.getElementsByClassName("name-input");
let email = document.getElementsByClassName("email-input");
let text = document.getElementsByClassName("text-input");

document.addEventListener('DOMContentLoaded', (event) => {
    let savedContact = JSON.parse(infoString);
    name.value = savedContact.name;
    email.value = savedContact.email;
    text.value = savedContact.text;
});
```
The code above will run whenever `DOM` content is loaded and the input values will be updated with the values from the parsed data from `localStorage`. Generally, it is not a good idea to leave `console.log()` in your code because they can cause security issues among others.

Reply:
@cwaku 

please i am really confused...are you saying i should remove
const name = document.getElementById('name');
const email = document.getElementById('email');
const text = document.getElementById('message');

Reply:
@Therealadetunji 
Mine was only an example and you are free to keep your previous way of getting the form inputs as it works fine.